lib/ansible/plugins/inventory/nmap.py

```diff
@@ -70,11 +71,10 @@ class InventoryModule(BaseInventoryPlugin, Constructable, Cacheable):
     def __init__(self):
 
         self._nmap = None
-        for path in os.environ.get('PATH').split(':'):
-            candidate = os.path.join(path, 'nmap')
-            if os.path.exists(candidate):
-                self._nmap = candidate
-                break
+        try:
+            self._nmap = get_bin_path('nmap', True)
+        except ValueError as e:
```

Comment:
delay this to 'parse' and issue a 'missing nma binary' explanation there, we don't want plugins to fail on instanciation